verification/src/block_verifier.rs

```diff
@@ -133,15 +133,15 @@ impl MerkleRootVerifier {
 
     pub fn verify(&self, block: &Block) -> Result<(), Error> {
         if block.header().transactions_root() != &block.cal_transactions_root() {
-            return Err(Error::CommitTransactionsRoot);
+            Err(BlockError::UnmatchedCommittedRoot)?;
```

Comment:
TransactionRoot?

Reply:
I am not sure about it. But I think it would be better to specify which type of transactions in error messages, proposal or committed (`DuplicatedCommittedTransactions/DuplicatedProposalTransactions`, `UnmatchedCommittedRoot/UnmatchedProposalRoot`)